lib/hsa/mcwamp_hsa.cpp

```diff
@@ -77,10 +77,7 @@
 // If limit is exceeded, HCC will force a queue wait to reclaim
 // resources (signals, kernarg)
 // MUST be a power of 2.
-#define MAX_INFLIGHT_COMMANDS_PER_QUEUE  (2*8192)
-
-// threshold to clean up finished kernel in HSAQueue.asyncOps
-#define ASYNCOPS_VECTOR_GC_SIZE (2*8192)
+int HCC_MAX_INFLIGHT_COMMANDS_PER_QUEUE=(2*8192);
```

Comment:
@jeffdaily I'm not very fond of this less-than-desirable coding style introduced by @bensander ... ALL-CAPITAL global variables... but we'll leave it for now